pom.xml

```diff
@@ -115,9 +115,9 @@
                 <version>1.0.0</version>
             </dependency>
             <dependency>
-                <groupId>com.autonomy.aci.client</groupId>
-                <artifactId>autonomy-aci-client-api</artifactId>
-                <version>4.1.3</version>
```

Comment:
Update to the next major version - as this is dependency management, dropping this is a breaking change.

Reply:
downgrading to 5.0.0

Reply:
How does that help?